### PR TITLE
Introduce file getter view

### PR DESCRIPTION
**Pull request by bierik**
Took the implementation from https://github.com/4teamwork/mycolex/commit/d7b861a89433b4242ce0f06dfb847a7114d6de75

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,3 +149,82 @@ Loads given datasets using `python manage.py load_e2e_data --datasets [datasets]
...
+from django_utils.views import FileGetterView
+
+class MyFileView(FileGetterView):
+    def get_object(self, model_class, id, request, **kwargs):
+        return model_class._default_manager.get(user=request.user, id=id)
+```
+
+### URL
+
+To relay all the requests for assets to the file getter view, append your urls with the a pattern like this:
```
Comment (lvonlanthen) [inline README.md:184]:
```suggestion
To relay all the requests for assets to the file getter view, append your urls with a pattern like this:
```

File: README.md [hunk 2]
```diff
@@ -149,3 +149,82 @@ Loads given datasets using `python manage.py load_e2e_data --datasets [datasets]
...
+
+Your pattern has to end with `(?P<file_info>.*)` and register a file getter view.
+
+### Serializer
+
+In order for the file getter view to resolve certain model fields, the field must be a `ProtectedImageField` in the serializer:
+
+```python
+class MySerializer(Serializer):
+    image = ProtectedImageField()
```
Comment (lvonlanthen) [inline README.md:200]: Could you please add the import for this `ProtectedImageField` in the example?

**Review by lvonlanthen — APPROVED** [review]
Some nitpicks, otherwise 🚀

File: README.md (hunk 1 above)
Comment (bierik) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (bierik) [reply README.md:202]: done

**Pull request merged**